README.md

```diff
@@ -111,6 +111,40 @@ GovukStatsd.gauge "bork", 100
 GovukStatsd.time("account.activate") { @account.activate! }
 ```
 
+## Healthchecks
+
+Set up a route in your rack-compatible Ruby application, and pick the built-in
+or custom checks you wish to perform.
+
+Custom checks must be any object which responds to `#run`.
+
+For Rails apps:
+```ruby
+module CustomCheck
+  def self.run
+    {
+      my_custom_check: {
+        status: ThingChecker.everything_okay? ? "OK" : "Critical"
+      }
+    }
+  end
+end
+```
+```ruby
+require "govuk_app_config/govuk_healthcheck"
+
+get "/healthcheck", to: GovukHealthcheck.rack_response(
```

Comment:
Would it be worth it to set up an engine so we can do something like this?

```rb
mount GovukAppConfig::Engine, at: "/healthcheck"
```

Reply:
Maybe, though perhaps not in the first pass.  Right now there's not really much to wrap.